### ISSUE TITLE
Problem with user dropdown menu

### ISSUE THREAD

**Issue by STLawson** [melvingb/we_universal#2]
When a user is logged in and they try to click the dropdown menu in the top right of the page, it takes them to the User Control Panel rather than dropping down a menu of options (including logout).

For testing purposes, here is the site that I'm using it on: http://www.leathernecknation.net/forum/
I have set up a test user account under the following....
username: testuser
password: testlogin
(don't worry, super-restrictive permissions and I'll delete the account when I'm done with it)
But it'll let you log in and see what I'm talking about (if you are otherwise unaware of the issue).

Thanks for your help.

**Comment by PeopleInside** [comment]
Hi,
I AM using we_universal released from nextgen and I can ensure the issue you are reporting is not present.
Maybe is some customization done by you. I tested also the log in into your forum and I was able to see the issue. Tested on mine and issue is not present :-) so maybe nextgen can be busy and cannot help you with this specific issue as you seems customized the theme.

**Comment by STLawson** [comment]
Ok, I'll have to take a look at that then. I did do some customization, but I didn't think anything I was doing would be affecting the dropdown. I'll dig back in and maybe disable my mods too, to see if anything there might be affecting it.

**Comment by melvingb** [comment]
Hi @STLawson, what extension you using?

**Comment by STLawson** [comment]
Enabled extensions are: Advanced BBCode Box, Ajax Chat, Google Analytics, Lightbox, Pages.

I did disable all my extensions and tested that and still had the problem.

I have also reloaded the base prosilver style as well as this version of the we_universal style, and done a file comparison between the base phpbb files and my install to make sure there wasn't anything odd there. Still had the same problem though. 

My next step is to do a fresh install of phpbb in another directory, tie it to my existing databases, then go through my modifications step-by-step to see what is causing the issue, unless you have some other suggestion(s).

I'm not an expert, but I do know html, css, and php fairly well, so if you'd like, I can report back if I am able to discover what the issue is...and if it has anything to do with something in the we_universal style or just something else unrelated that is messing things up.

UPDATE - did a fresh install of phpbb on a second directory, installed my modified version of we_universal, and the dropdown still works. So now I'm copying over a fresh phpbb over my old phpbb files to see if it's something in there that I screwed up. I'll let you know, but at this point, it doesn't look like a style issue (or even an issue with my modded version of the style, just fyi.

UPDATE 2 - never really figured out what the issue was, I must have accidentally modified one of the phpbb core template files rather than the one in the style directory. After copying over fresh phpbb files and installing my modified we_universal theme, it worked. Feel free to delete this issue thread if you feel it's not relevant (since the issue wasn't with we_universal itself....just my own glitch).
